clog/builder.go

```diff
@@ -101,6 +121,17 @@ func (b builder) log(l logLevel, msg string) {
 	case LevelError:
 		zsl.Error(msg)
 	}
+
+	// add otel logging if provided
+	otelLog := b.otel
+
+	if otelLog == nil {
+		otelLog = cluesNode.OTELLogger()
+	}
+
+	if otelLog != nil {
+		otelLog.Emit(b.ctx, record)
```

Comment:
Assuming we'll update alcion data plane with latest clues at some point, I am a little vary of turning this on by default. Have we evaluated whether this can cause any perf impact? Also, have we tested if initializing an otel endpoint inside ecs works? If not, I'd prefer we make otel hook-in optional. What do you think?

Reply:
Ah I am guessing we won't call `clues.Initialize` in alcion data plane, so otel endpoint will never be stood up.

Reply:
Please confirm.

Reply:
> Assuming we'll update alcion data plane with latest clues at some point

Those updates can be frozen and/or use the (now archived) clog repo, instead of the clues repo.  There's really no reason to bump the package with us on maintenance mode.  Exception cases can be handled as they arise.

> guessing we won't call clues.Initialize in alcion data plane

In the case that we do bump clues for alcion, yes, that's definitely the easy way out.

Reply:
Got it, thanks!